projects/docs/docs/commands/test.md

```diff
@@ -50,11 +50,12 @@ One of the benefits of using Tuist over other automation tools is that developer
 
 ### Arguments
 
-| Argument            | Short | Description                                                         | Default           | Required |
-| ------------------- | ----- | ------------------------------------------------------------------- | ----------------- | -------- |
-| `--clean`           | n/a   | `When passed, it cleans the project before testing it.`             | False             | No       |
-| `--path`            | `-p`  | `The path to the directory that contains the project to be tested.` | Current directory | No       |
-| `--device`          | `-d`  | `Test on a specific device.`                                        |                   | No       |
-| `--os`              | `-o`  | `Test with a specific version of the OS.`                           |                   | No       |
-| `--configuration`   | `-C`  | `The configuration to be used when building the scheme.`            |                   | No       |
-| `--skip-ui-tests`   | n/a  | `When passed, it skips testing UI Tests targets.`                   | False             | No       |
+| Argument               | Short | Description                                                         | Default           | Required |
+| ---------------------- | ----- | ------------------------------------------------------------------- | ----------------- | -------- |
+| `--clean`              | n/a   | `When passed, it cleans the project before testing it.`             | False             | No       |
```

Comment:
Same here, I thin we can make `-c` the short version for this one. It's not related to your PR, but we can take the opportunity and extend it.